cpp/src/parquet/encoding.h

```diff
@@ -394,6 +394,12 @@ class DictDecoder : virtual public TypedDecoder<DType> {
 // ----------------------------------------------------------------------
 // TypedEncoder specializations, traits, and factory functions
 
+class BooleanDecoder : virtual public TypedDecoder<BooleanType> {
+ public:
+  using TypedDecoder<BooleanType>::Decode;
+  virtual int Decode(uint8_t* buffer, int max_values) = 0;
```

Comment:
Can you add a docstring? It should be clear that `buffer` will contain bit-packed values.

Reply:
Done. Added a docstring.